ssh-key/src/signature.rs

```diff
@@ -517,6 +519,15 @@ impl_signature_for_curve!(p256, "p256", NistP256, 32);
 impl_signature_for_curve!(p384, "p384", NistP384, 48);
 impl_signature_for_curve!(p521, "p521", NistP521, 66);
 
+#[cfg(any(feature = "p256", feature = "p384", feature = "p521"))]
+fn build_field_bytes<B: FromIterator<u8> + Copy>(m: Mpint) -> Option<B> {
+    let bytes = m.as_positive_bytes()?;
+    #[allow(unused_qualifications)] // size_of is in the prelude of Rust > 1.80
```

Comment:
Alternatively you can import it as `core::mem::size_of`

Reply:
I _though_ CI got mad about that as well, but let's check... (baa16c27ecf03a44811fbbde7ce74e49307c9903)